feed/views.py

```diff
@@ -13,3 +14,13 @@ def feed(request):
 
 class PostDetailView(DetailView):
     model = Post
+
+
+class PostCreateView(LoginRequiredMixin, CreateView):
+    model = Post
+    fields = ['title', 'content']
+    login_url = '/login/'
+
+    def form_valid(self, form):
```

Comment:
Look at the samples above, there is recommended way to do it

Reply:
Done